client/options.go

```diff
@@ -247,6 +248,110 @@ func IsBackup() BulkInsertOption {
 	}
 }
 
+func WithBucket(bucket string) BulkInsertOption {
+	return func(req *milvuspb.ImportRequest) {
+		optionMap := entity.KvPairsMap(req.GetOptions())
+		optionMap["bucket"] = bucket
```

Comment:
bucketName

Reply:
params.add("address", "localhost")
params.add("port", 9000)
params.add("accessKeyID", "minioadmin")
params.add("secretAccessKey", "minioadmin")
params.add("useSSL", False)
params.add("bucketName", "a-bucket")
params.add("rootPath", "files")
params.add("useIAM", False)
params.add("cloudProvider", "aliyun")
params.add("iamEndpoint", "")
params.add("region", "")
params.add("useVirtualHost", True)

Reply:
bucket is an already existing name, it can't be compatible with old version if we change it